src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -192,13 +192,12 @@ private void initializeCommandGroups() {
 
       twoBallCommand =
           new SequentialCommandGroup(
-              new DeployIntake(intakeSubsystem, cdsSubsystem),
+              // new DeployIntake(intakeSubsystem, cdsSubsystem),
```

Comment:
I think we still want to deploy the intake in autonomous, was this just for testing?

Reply:
The parallel command that has combined intake and cds will deploy it.